idb/postgres/postgres_integration_test.go

```diff
@@ -1549,7 +1550,7 @@ func TestAddBlockCreateDeleteAppSameRound(t *testing.T) {
 	defer l.Close()
 
 	appid := uint64(1)
-	createTxn := test.MakeCreateAppTxn(test.AccountA)
+	createTxn := test.MakeCreateSimpleAppTxn(test.AccountA)
```

Comment:
renaming because a slightly more complex function is introduced rendering the original as "Simple"